### PR TITLE
Rename and move script-run to file-run

### PR DESCRIPTION
**Pull request by Ordinant**
plus changed a few code-block:: none to code-block:: text. (See Document-748) ;-}

### CODE REVIEW

**Comment by Ordinant** [comment]
@anusudarsan, please review and approve
@mosabua, FYI

File: docs/presto-admin-commands.rst [hunk 1]
```diff
@@ -374,11 +374,31 @@ defaults to /tmp.
...
+Use this command to run an arbitrary script on a cluster. It copies the script
+from its local location to the specified remote directory (defaults to /tmp),
+makes the file executable, and runs it.
+
+Example
+-------
+
+.. code-block:: text
+
+    ./presto-admin script run /my/local/script.sh
```
Comment (anusudarsan) [inline docs/presto-admin-commands.rst:399]: file run here

Comment (Ordinant) [reply docs/presto-admin-commands.rst]: OK, fixed with **git commit --amend** so that it still looks like one commit.

**Review by anusudarsan — APPROVED** [review]

**Comment by anusudarsan** [comment]
Merged. thanks

**Pull request merged**